.circleci/config.yml

```diff
@@ -106,7 +106,7 @@ jobs:
       - run:
           name: Docker push
           command: |
-            $(aws ecr get-login --no-include-email --region ap-northeast-1)
+            aws ecr get-login-password --region ap-northeast-1 | docker login --username AWS --password-stdin https://564677439943.dkr.ecr.ap-northeast-1.amazonaws.com
```

Comment:
> __[Secret Scan]__ It may be dangerous to commit the AWS account ID. ([view](https://sider.review/gh/repos/50192141/pulls/1824#issue-7152242))
>
> <details>
>
> AWS account ID is a credential (12-digit number). Once this leaks, an attacker may identify essential information
> on your AWS account, such as roles, users, etc.
>
> See also:
> * https://docs.aws.amazon.com/general/latest/gr/acct-identifiers.html
> * https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html
> * https://rhinosecuritylabs.com/aws/assume-worst-aws-assume-role-enumeration/
>
> | Rule | Severity |
> | ---- | -------- |
> | `sider.secrets.aws.account_id` | `warning` |
>
> You can [close this issue](https://sider.review/gh/repos/50192141/pulls/1824#issue-7152242) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue. Reason:
- When this is not an AWS account ID.